tests/unit/classes/column-test.js

```diff
@@ -29,6 +29,23 @@ test('create column - column instance', function(assert) {
   assert.equal(col2.label, 'Name');
 });
 
+test('reopen column', function(assert) {
+  Column.reopenClass({
+    someStaticMethod() {
+      return true;
+    }
+  });
+  assert.ok(Column.someStaticMethod());
+
+  Column.reopen({
+    someMethod() {
+      return true;
+    }
+  });
+  let col = new Column();
+  assert.ok(col.someMethod());
+});
+
```

Comment:
I think, I should rather write a specialized unit test for `utils/fix-proto`. This way, there's no mutation of global classes.